README.md

```diff
@@ -144,6 +144,30 @@ prom-label-proxy \
 
 `prom-label-proxy` will enforce the `tenant=~"prometheus|alertmanager"` label selector in all requests.
 
+You can match label value  using a regular expression with the `-regex-match` option. For example:
+
+```
+prom-label-proxy \
+   -header-name X-Namespace \
+   -label namespace \
```

Comment:
(suggestion) I'd use a static label value to make it more "visible".
```suggestion
   -label-value '^foo-.+$' \
   -label namespace \
```

Reply:
I wonder if we shouldn't anchor the regexp by default (like Prometheus does for relabelings)? It would at least prevent too-permissive configurations.
It'd also be good to require that the compiled regex doesn't match the empty string.

Reply:
> I wonder if we shouldn't anchor the regexp by default (like Prometheus does for relabelings)? It would at least prevent too-permissive configurations.

Prometheus anchors already.

> It'd also be good to require that the compiled regex doesn't match the empty string.

Good idea.